scalikejdbc-core/src/main/scala/scalikejdbc/TypeBinder.scala

```diff
@@ -22,84 +22,58 @@ trait TypeBinder[+A] {
  * Type binder for java.sql.ResultSet.
  */
 object TypeBinder extends LowPriorityTypeBinderImplicits with UnixTimeInMillisConverterImplicits {
+  // TODO: Now, TypeBinder does not need UnixTimeInMillisConverterImplicits.
```

Comment:
// TODO: TypeBinder object actually doesn't need UnixTimeInMillisConverterImplicits. Remove UnixTimeInMillisConverterImplicits in 2.5. Since removing it breaks bin-compatibility, we cannot do that in 2.4 series.